code/game/gamemodes/clock_cult/clock_objects.dm

```diff
@@ -556,17 +556,16 @@
 	var/new_obj_type //The path of the new type of object to replace the old
 	var/alloy_cost = 0
 	var/valid_target = FALSE //If the proselytizer will actually function on the object
-	switch(target.type)
-		if(/turf/closed/wall, /turf/closed/wall/rust)
-			operation_time = 50
-			new_obj_type = /turf/closed/wall/clockwork
-			alloy_cost = 10
-			valid_target = TRUE //Need to change valid_target to 1 or TRUE in each check so that it doesn't return an invalid value
-		if(/turf/open/floor/plating, /turf/open/floor/plasteel)
-			operation_time = 30
-			new_obj_type = /turf/open/floor/clockwork
-			alloy_cost = 5
-			valid_target = TRUE
+	if(istype(target, /turf/closed/wall))
```

Comment:
Ghetto 'break through walls' tool for servants, also being able to convert enemy cult walls when they can't do so back seems a little silly